### ISSUE TITLE
Sunset SystemContractCache

### ISSUE THREAD

**Issue by xcthulhu** [casper-network/casper-node#1699]

**Comment by goral09** [comment]
This is a very big change, if you create a ticket for removal of it you should also include the explanation as to why.

**Comment by cspramit** [comment]
@darthsiroftardis is there another ticket or PR for this? seems we may have tackled this already

**Comment by darthsiroftardis** [comment]
Yes, this one @cspramit https://app.zenhub.com/workspaces/engineering-60953fafb1945f0011a3592d/issues/casper-network/casper-node/2152
